src/gsi/gsi_rfv3io_mod.f90

```diff
@@ -2932,11 +2937,18 @@ subroutine gsi_fv3ncdf_read_ens_parallel_over_ens(filenamein,fv3filenamegin, &
              varname_files = (/'sphum',' o3mr'/)
           end if
        end if
-       if( present(dbz) )then            ! phyvars: dbz
+       if( present(dbz) .and. present(fed) )then  ! phyvars: dbz, fed
+          allocate(varname_files(2))
+          varname_files = (/'ref_f3d','flash_extent_density'/)
```

Comment:
Here need to be:
varname_files = (/'ref_f3d             ','flash_extent_density'/)
for gcc compiler to pass.

Reply:
@hu5970 
One blank space is enough? Thanks. 
Hongli

Reply:
No, it needs 13 blank spaces to make it equals to the length of "flash_extent_density".

Reply:
Thanks, Ming!